vpn-router-web/src/main/java/vpnrouter/web/model/Client.java

```diff
@@ -23,17 +22,33 @@ public class Client {
     @With
     private final boolean tunnelled;
 
-    public static List<Client> from(List<ClientView> clients) {
-        return clients.stream()
-                .map(Client::from)
-                .toList();
-    }
-
     public static Client from(ClientView client) {
         return OBJECT_MAPPER.convertValue(client, Client.class);
     }
 
     public ClientUpdate toClientUpdate() {
         return OBJECT_MAPPER.convertValue(this, ClientUpdate.class);
     }
+
+    public ClientCreation toClientCreation() {
+        return OBJECT_MAPPER.convertValue(this, ClientCreation.class);
+    }
+
+    @Getter
+    @Setter
+    public static class Wrapper {
+        private String ipAddress;
+        private String name;
+        private boolean tunnelled;
+
+        public Client build() {
+            return new Client(ipAddress, name, tunnelled);
+        }
+    }
+
+    public static List<Client> from(List<ClientView> clients) {
```

Comment:
No 🤦 This is not a class, this is a method.

Why don't you just use Cmd+Option+L?

Reply:
I use Cmd + Option + L but it doesn't move methods inside a class) So I did it manually

Reply:
You should perform Cmd+Option+L, having selected a project root folder in your tree view (in the side bar on the left).

Reply:
And also, having selected 'Rearrange entries' checkbox

Reply:
Wow! It really works))